sites/automationworld/redirect-handler.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  check: (from) => {
+    // Digital Edtion
+    let pattern = /\/sites\/default\/files\/digital_edition\/(.*)/;
+    let matches = pattern.exec(from);
+    if (matches) {
```

Comment:
Check for matches and matches[1]:
`if (matches && matches[1]) {`

This applies for all `matches` checks